desktop/cmp/dash/DashContainerModel.js

```diff
@@ -114,6 +122,7 @@ export class DashContainerModel {
         viewSpecs,
         initialState = [],
         showAddButton = true,
+        addViewContent = () => dashContainerAddViewPanel(),
```

Comment:
This can be specified as just dashContainerAddViewPanel.  (no need for additional wrapping function).

Reply:
Probably can be called just AddViewPanel as well, if it's an implementation class.